packages/safe-tools-client/app/components/future-payments-list/index.css

```diff
@@ -0,0 +1,29 @@
+.future-payments-list {
+  min-width: 20rem;
+  max-width: 25rem;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.no-payments-section {
+  height: 100%;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.no-payments-section .no-payments-title {
+  font: 700 1.125rem/calc(24 / 18) var(--boxel-font-family);
```

Comment:
Where is this rem calculation coming from?

Reply:
I got it from this class `.boxel-action-container-section-title`